src/braket/circuits/angled_gate.py

```diff
@@ -75,8 +72,7 @@ def parameters(self) -> list[Union[FreeParameterExpression, float]]:
 
     @property
     def angle(self) -> Union[FreeParameterExpression, float]:
-        """
-        Returns the angle for the gate
+        """Returns the angle for the gate
```

Comment:
```suggestion
        """Returns the angle of the gate
```
In fact, does a `property` even need a summary? It's not included for many other `property` methods